src/validators/custom.js

```diff
@@ -0,0 +1,19 @@
+function custom(value, options) {
+  const keys = Object.keys(options)
```

Comment:
With the validations this will just execute the commands, why not to enable options as the other validations?

With that we will could insert custom functions and possibilities to customize in a high level, without change much and enable other functions as plugins or extensions of this lib...

This is just a suggestion, what you think about this?